examples/layers/rendering/src/sector_layout.dart

```diff
@@ -14,14 +14,14 @@ class SectorConstraints extends Constraints {
     this.maxDeltaRadius = double.infinity,
     this.minDeltaTheta = 0.0,
     this.maxDeltaTheta = kTwoPi,
-  }) : assert(maxDeltaRadius >= minDeltaRadius),
-       assert(maxDeltaTheta >= minDeltaTheta);
+  })  : assert(maxDeltaRadius >= minDeltaRadius),
```

Comment:
Why insert an extra space here and below?

Reply:
Not completely sure but IIRC it allow to have initializer elements always on the same column and on a column that is a multiple of 2. A lot of editors (especially VSCode) use 2-space indent when you use tab or when you shift a block of code with tab/maj+tab.